sys/Makefile.dep

```diff
@@ -901,12 +495,12 @@ ifneq (,$(filter gcoap,$(USEMODULE)))
   USEMODULE += sock_util
   USEMODULE += event_callback
   USEMODULE += event_timeout
-ifneq (,$(filter gnrc%,$(USEMODULE)))
-  USEMODULE += gnrc_sock_async
-endif
-ifneq (,$(filter openwsn%,$(USEMODULE)))
-  USEMODULE += openwsn_sock_udp
-endif
+  ifneq (,$(filter gnrc%,$(USEMODULE)))
+    USEMODULE += gnrc_sock_async
+  endif
```

Comment:
This might also be movable to the new gnrc-specific Makefile.dep:

```Makefile
ifneq (,$(filter gcoap,$(USEMODULE)))
  ifneq (,$(filter gnrc%,$(USEMODULE)))
    USEMODULE += gnrc_sock_async
  endif
endif
```

Reply:
> If it would be renamed to `auto_init_netif_gnrc` or `gnrc_auto_init_netif` it would work. But with `gnrc` in the middle, GNRC's `Makefile.dep` will not be included (unless by chance another module with a name starting or ending with gnrc is used). I'll add the comment.

Could this become a problem here as well?